app/views/shared/_post_card.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="card">
   <div class="card-body">
     <h4 class="card-title"><%= post.title %></h4>
-    <div class="text-truncate text-justify"><%= post.text %></div>
+    <div class="text-truncate text-justify"><%= post.excerpt %></div>
```

Comment:
- [ ] About 2/3 of your tests are still failing. Kindly evaluate and fix this.

  ![image](https://github.com/Bifabig/blog-app-rails/assets/54848363/fa92fd93-35ee-427d-a10a-17686cfb1bb0)

  Here is an example of a failed test. Most of them fail due to this "excerpt" here in the post partial which does not exist, according to the error.

  ![image](https://github.com/Bifabig/blog-app-rails/assets/54848363/2d0bc33a-8ed3-4780-9ac1-c1a0ace22612)

Reply:
Thanks @DammyShittu